Tests/UINotificationCenterTests.swift

```diff
@@ -8,9 +8,17 @@
 
 import XCTest
 @testable import UINotifications
+import ConcurrencyExtras
 
+@MainActor
 final class UINotificationCenterTests: UINotificationTestCase {
     
+    override func invokeTest() {
+        withMainSerialExecutor {
```

Comment:
This is from the `ConcurrencyExtras` library and ensures all tests run in serial. See: https://pointfreeco.github.io/swift-concurrency-extras/main/documentation/concurrencyextras/reliablytestingasync